### ISSUE TITLE
daml-on-sql docs should say what information is logged and at what levels.

### ISSUE THREAD

**Issue by bame-da** [digital-asset/daml#7206]
The current docs page has extensive information on metrics, but little on logs. This was pointed out in https://discuss.daml.com/t/is-there-any-metrics-related-to-ledger-api-connections/1072/5.

**Comment by SamirTalwar-DA** [comment]
I agree. However, logging is in a pretty bad state when compared to metrics. In order to document them, we'd first have to _have_ logs.

This is captured in a few issues, including #6684.

**Comment by mziolekda** [comment]
Obsolete issue
